spec/jobs/mailchimp_nonprofit_user_job_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe MailchimpNonprofitUserJob, type: :job do
+  let(:user) {create(:user)}
+  let(:nonprofit) {create(:nonprofit_base)}
+  let(:drip_email_list) {create(:drip_email_list_base)}
+
+  it 'enqueues job when nonprofit user signed up' do 
+    expect(Mailchimp).to receive(:signup).with(user, nonprofit)
```

Comment:
This seems to be testing that `Mailchimp.signup` is called. But we're not supposed to be testing that, we're supposed to be testing that `Mailchimp.signup_nonprofit_user` is called.